src/index.js

```diff
@@ -1,21 +1,25 @@
 import _ from 'lodash';
 import './style.css';
-import { sortList } from './data.js';
+import { add, loadList } from './data.js';
 
-const ul = document.getElementById('leaderboard');
+const gameId = 'S58ZtGSMOy7mSGqc3Q6H';
+const myList = document.getElementById('leaderboard');
 const form = document.getElementById('entry-form');
+const btnRefresh = document.getElementById('btnRefresh');
 
 form.addEventListener('submit', (event) => {
   event.preventDefault();
+  const users = {};
+  users.user = form.elements.name.value;
+  users.score = form.elements.score.value;
+  add(gameId, users, myList);
+});
```

Comment:
[OPTIONAL] Good job hitting the api :clap: , I think it will be better if we clear the form data once user submits the form with valid input, this will make sure the users do not submit same data multiple times by clicking the sublit button again and again and it will also acts as a confirmation that the data is submitted :+1:
here is a example of what I am talking about : 
```
const clearForm = () => {
  form.elements.name.value = '';
  form.elements.score.value = '';
};

```
then we add the clearform() after adding the score

Reply:
Thanks for your suggestion. I have added that function.